he_twitter_miner/src/twitter-sample-stream.json

```diff
@@ -0,0 +1,29 @@
oto\/1","type":"photo","sizes":{"small":{"w":680,"h":510,"resize":"fit"},"thumb":{"w":150,"h":150,"resize":"crop"},"medium":{"w":922,"h":692,"resize":"fit"},"large":{"w":922,"h":692,"resize":"fit"}},"source_status_id":852359457065627648,"source_status_id_str":"852359457065627648","source_user_id":492652749,"source_user_id_str":"492652749"}]},"extended_entities":{"media":[{"id":852359446999314432,"id_str":"852359446999314432","indices":[23,46],"media_url":"http:\/\/pbs.twimg.com\/media\/C9QwZqtXsAANunO.jpg","media_url_https":"https:\/\/pbs.twimg.com\/media\/C9QwZqtXsAANunO.jpg","url":"https:\/\/t.co\/Ncs684Bb4U","display_url":"pic.twitter.com\/Ncs684Bb4U","expanded_url":"https:\/\/twitter.com\/ironicovcs\/status\/852359457065627648\/photo\/1","type":"photo","sizes":{"small":{"w":680,"h":510,"resize":"fit"},"thumb":{"w":150,"h":150,"resize":"crop"},"medium":{"w":922,"h":692,"resize":"fit"},"large":{"w":922,"h":692,"resize":"fit"}},"source_status_id":852359457065627648,"source_status_id_str":"852359457065627648","source_user_id":492652749,"source_user_id_str":"492652749"}]},"favorited":false,"retweeted":false,"possibly_sensitive":false,"filter_level":"low","lang":"pt"},"is_quote_status":false,"retweet_count":0,"favorite_count":0,"entities":{"hashtags":[],"urls":[],"user_mentions":[{"screen_name":"ehdaora","name":"lindu","id":478072755,"id_str":"478072755","indices":[3,11]}],"symbols":[],"media":[{"id":852359446999314432,"id_str":"852359446999314432","indices":[36,59],"media_url":"http:\/\/pbs.twimg.com\/media\/C9QwZqtXsAANunO.jpg","media_url_https":"https:\/\/pbs.twimg.com\/media\/C9QwZqtXsAANunO.jpg","url":"https:\/\/t.co\/Ncs684Bb4U","display_url":"pic.twitter.com\/Ncs684Bb4U","expanded_url":"https:\/\/twitter.com\/ironicovcs\/status\/852359457065627648\/photo\/1","type":"photo","sizes":{"small":{"w":680,"h":510,"resize":"fit"},"thumb":{"w":150,"h":150,"resize":"crop"},"medium":{"w":922,"h":692,"resize":"fit"},"large":{"w":922,"h":692,"resize":"fit"}},"source_status_id":852359457065627648,"source_status_id_str":"852359457065627648","source_user_id":492652749,"source_user_id_str":"492652749"}]},"extended_entities":{"media":[{"id":852359446999314432,"id_str":"852359446999314432","indices":[36,59],"media_url":"http:\/\/pbs.twimg.com\/media\/C9QwZqtXsAANunO.jpg","media_url_https":"https:\/\/pbs.twimg.com\/media\/C9QwZqtXsAANunO.jpg","url":"https:\/\/t.co\/Ncs684Bb4U","display_url":"pic.twitter.com\/Ncs684Bb4U","expanded_url":"https:\/\/twitter.com\/ironicovcs\/status\/852359457065627648\/photo\/1","type":"photo","sizes":{"small":{"w":680,"h":510,"resize":"fit"},"thumb":{"w":150,"h":150,"resize":"crop"},"medium":{"w":922,"h":692,"resize":"fit"},"large":{"w":922,"h":692,"resize":"fit"}},"source_status_id":852359457065627648,"source_status_id_str":"852359457065627648","source_user_id":492652749,"source_user_id_str":"492652749"}]},"favorited":false,"retweeted":false,"possibly_sensitive":true,"filter_level":"low","lang":"pt","timestamp_ms":"1496265366657"}
```

Comment:
in case you want to play around with the raw data, its only a couple of dozen rows, maybe for unit testing